package.json

```
@@ -93,13 +93,13 @@
         "icon": "$(add)"
       },
       {
-        "command": "onevscode.flip-cfg-to-gui",
+        "command": "one.editor.open",
```

If we introduce new type of custom editor, we cannot distinguish from it.
So I think adding `cfg` word seems better :)

```suggestion
        "command": "one.editor.openCfg",
```

---

Do we plan to? I thought we plan to add more backend but not a cfg editor..

---

What about renaming it later then? :-D